zone/client.cpp

```diff
@@ -4269,6 +4269,32 @@ void Client::KeyRingList()
 	}
 }
 
+bool Client::IsLevelFirst(uint32 p_race, uint32 p_class, uint16 level) {
+
+	std::string query = StringFormat("SELECT count(*) FROM server_first_levels WHERE level = '%lu' AND race = '%lu' AND class = '%lu'", level, p_race, p_class);
```

Comment:
Can use fmt::format here instead, refer to [this](https://github.com/EQEmu/Server/pull/1326/files#diff-e5771b638ab703fb34c3af79eff05343daac5c3742a5edc37e0257022b0d8cbaR4898).